### PR TITLE
[274] Add Tailwinds via Webpack

### LINKED ISSUE #274: Pick a CSS framework and implement it

### PR DESCRIPTION
**Pull request by megantrimble**
# Checklist:

- [x] I have performed a self-review of my own code,
- [x] I have commented my code, particularly in hard-to-understand areas,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works,
- [x] New and existing unit tests pass locally with my changes ("bundle exec rake"),
- [x] Title include "WIP" if work is in progress.

Resolves #274 

### Description
Installed Tailwindcss via yarn. Tailwinds was added as a PostCSS plugin. A tailwind.js and styles.scss file was included under packs. Bootstrap import was removed. This will allow developers to easily customize the CSS.

Dependencies:
- Tailwindcss [https://tailwindcss.com/docs/installation]

### Type of change

* New feature (non-breaking change which adds functionality)
 - This does break minor styles with no effect to functionality (ie table width and button placement). Follow up PR's will include fixes for this.

### How Has This Been Tested?

Checked all buttons and links to verify that they are working and all pages still include full functionality.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 @import "facilities";
 @import "file_uploads";
 @import "home";
 @import "sessions";
 @import "variables";
 @import "mixins";
 @import "pages/**/*";
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
```
Comment (wadewinningham) [inline app/assets/stylesheets/application.scss:10]: Are these necessary with them being set in `app/javascript/packs/styles.scss` ?

Comment (megantrimble) [reply app/assets/stylesheets/application.scss]: They are not. Thanks for the catch. Just removed them!

**Comment by CraigJZ** [comment]
At some point, we will probably want to remove the bootstrap gem from the Gemfile and popper +  bootstrap from `app/assets/javascripts/application.js` but that isn't blocking for this PR.

**Review by CraigJZ — APPROVED** [review]
Thank you!!!!!!

**Comment by haydenrou** [comment]
🙌

**Pull request merged**